src/early/ipfs-script.js

```diff
@@ -0,0 +1,30 @@
+import fs from 'fs-extra'
+import { join } from 'path'
+import os from 'os'
+import { logger } from '../utils'
+
+export default function () {
+  // Note: during runtime, we only do this for darwin.
+  if (os.platform() !== 'darwin') {
+    return
+  }
+
+  // Ignore during development because the paths are not the same.
+  if (process.env.NODE_ENV === 'development') {
+    logger.info('[ipfs on path] unavailable during development')
+    return
+  }
+
+  // Don't make any changes if IPFS already exists...
+  if (fs.existsSync('/usr/local/bin/ipfs')) {
```

Comment:
`install.sh` from a go-ipfs distribution will try to install in either `/usr/local/bin` or `/usr/bin` https://github.com/ipfs/go-ipfs/blob/622dbf7f266e513c6de00cddc5cb851aa5cbfe61/cmd/ipfs/dist/install.sh#L9

perhaps we should try and exec `ipfs` here to see if it exists on the users path.